.github/workflows/go-tests.yml

```diff
@@ -41,6 +62,8 @@ jobs:
           sudo apt-get install openjdk-8-jre-headless
 
       - name: Run tests
+        # Allow tests to fail pre-release Go versions
+        continue-on-error: ${{ matrix.prerelease }}
```

Comment:
If a test does fail for pre-release, what happens next? Is it just logged?

Reply:
it's just logged in the workflow outputs like below:
![Screen Shot 2021-01-12 at 4 45 41 pm](https://user-images.githubusercontent.com/37109549/104274766-b1b3bf80-54f5-11eb-9f04-2ab68f4a975d.png)

Reply:
It's not obvious from the output that the failure is for a pre-release version. Can that be annotated somewhere?

Reply:
As discussed offline I've updated the labels so that pre-release versions are specified explicitly. e.g.
![image](https://user-images.githubusercontent.com/37109549/104409264-fe60ce80-55b9-11eb-91b6-0be8383cdeca.png)